src/lib_ccx/params.c

```diff
@@ -2274,22 +2313,22 @@ int parse_parameters(struct ccx_s_options *opt, int argc, char *argv[])
 			opt->hauppauge_mode = 1;
 			continue;
 		}
-		if (strcmp(argv[i], "-mp4vidtrack") == 0)
+		if (strcmp(argv[i], "-mp4vidtrack") == 0 || strcmp(argv[i], "--mp4vidtrack") == 0)
 		{
 			opt->mp4vidtrack = 1;
 			continue;
 		}
-		if (strstr(argv[i], "-unicode") != NULL)
+		if (strstr(argv[i], "-unicode") != NULL || strstr(argv[i], "--unicode") != NULL)
 		{
 			opt->enc_cfg.encoding = CCX_ENC_UNICODE;
 			continue;
 		}
-		if (strstr(argv[i], "-utf8") != NULL)
+		if (strstr(argv[i], "-utf8") != NULL || strstr(argv[i], "--utf8") != NULL)
```

Comment:
This is not needed since we're using strstr() instead of strcmp(), so strstr() would match both.

Reply:
Got it